README.md

```diff
@@ -1,13 +1,41 @@
 [![CircleCI](https://circleci.com/gh/aws/eks-charts.svg?style=svg)](https://circleci.com/gh/aws/eks-charts)
 
-## My Project
+## EKS Charts
 
-TODO: Fill this README out!
+Add the EKS repository to Helm:
 
-Be sure to:
+```sh
+helm repo add eks https://aws.github.io/eks-charts
+```
 
-* Change the title in this README
-* Edit your repository description on GitHub
+### App Mesh
+
+Create the `appmesh-system` namespace:
+
+```sh
+kubectl create ns appmesh-system
+```
+
+Apply the App Mesh CRDs:
+
+```sh
+kubectl apply -f https://raw.githubusercontent.com/aws/eks-charts/master/stable/appmesh-controller/crds/crds.yaml
+```
+
+Install the App Mesh CRD controller:
+
+```sh
+helm upgrade -i appmesh-controller eks/appmesh-controller --namespace appmesh-system
```

Comment:
So is namespace expected to always be passed?  Can we add a default namespace?

Reply:
The default namespace is `default` same as with kubectl. Helm v3 doesn't support namespace definitions and that's a good thing :) it uses the kubectl context for that.